src/replayPackets.ts

```
@@ -0,0 +1,171 @@
+        client.removeListener(name, listener)
+        resolve(data)
+      }
+      client.on(name, listener)
+    })
+  }
+
+  const playPackets = replayData.packets.filter(p => p.state === 'play')
+  for (const packet of playPackets) {
+    if (packet.isFromServer) {
+      writePacket(packet.name, packet.params)
+      await new Promise(resolve => setTimeout(resolve, packet.diff))
```

ESLint

[error] Return values from promise executor functions cannot be read.

(no-promise-executor-return)